.chloggen/kubeletstats_fix_defaults.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

```suggestion
change_type: breaking
```
Any time we change a metric to enabled by default we require that it's considered a breaking change.

More context: https://github.com/open-telemetry/opentelemetry-collector/issues/9663